.github/workflows/docker.yml

```
@@ -0,0 +1,74 @@
+name: Create and publish a Docker images
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
```

We need to exclude RC tags (e.g. `v4.0.0-RC1`. According to [the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) we would need:


```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+"
```

In the future we can add support for RC tags, but it's required that these tags won't be shortened to `{{major}}` and `{{major}}.{{minor}}`. If we can do it now, then fine, but it's not a requirement for the initial implementation.